server/tests/models/services/CollectionJobService.test.js

```diff
@@ -10,91 +10,63 @@ describe('CollectionJob Service Tests', () => {
         await sequelize.close();
     });
 
-    const randomIdGenerator = () => Math.random().toString(36).substring(7);
-
     it('should create a new CollectionJob with default status', async () => {
         await dbCleaner(async () => {
-            const id = randomIdGenerator();
             const collectionJob =
                 await CollectionJobService.createCollectionJob(
-                    { id, testPlanReportId: 2 },
+                    { testPlanReportId: 2 },
                     COLLECTION_JOB_ATTRIBUTES
                 );
-
-            expect(collectionJob.id).toEqual(id);
             expect(collectionJob.status).toEqual('QUEUED');
         });
     });
 
     it('should create a new CollectionJob with provided status', async () => {
         await dbCleaner(async () => {
-            const id = randomIdGenerator();
-            const status = 'IN_PROGRESS';
+            const status = 'CANCELLED';
```

Comment:
This change doesn't seem related to the refactoring. Is it still possible to create a collection job with the "IN_PROGESS" status?

More generally, I can't find a case (or think of one to be added in the near future) for creating a collection job with a status other than "QUEUED". Do you think we could remove the parameter in a follow-up patch?

Reply:
There are no limitations to the string that collection job can have as a `status`. Currently we are relying on the use of an enum throughout all of the code that sets statuses for CollectionJobs to ensure consistency. For additional safety, we could check to make sure the string passed into `createCollectionJob` is present in the enum. That would've surfaced this leftover from earlier work. This test doesn't care about the content of the status string. It was using a status "IN_PROGRESS" that we stopped using fairly early in development. While editing the relevant tests I took the opportunity to change the test statuses to strings that are currently used.

Reply:
Jeepers, I forgot about our move away from "IN_PROGRESS". Thanks for the thorough explanation! We can discuss the need for alternate values at creation time later.